crm_operating_unit/models/crm_lead.py

```diff
@@ -1,22 +1,14 @@
 # © 2015-19 ForgeFlow S.L. - Jordi Ballester Alomar
 # © 2015-17 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class CRMLead(models.Model):
     _inherit = "crm.lead"
 
-    @api.model
-    def _get_default_operating_unit(self):
-        team = self.env["crm.team"]._get_default_team_id()
-        if team.operating_unit_id:
-            return team.operating_unit_id
-        return self.env["res.users"].operating_unit_default_get(self._uid)
```

Comment:
`return self.env["operating.unit"]`

I think the default has to be filled in case team has one.